cmd/sanitize-prow-jobs/main.go

```diff
@@ -132,7 +199,14 @@ func main() {
 	}
 
 	if len(opt.prowJobConfigDir) > 0 {
-		if err := determinizeJobs(opt.prowJobConfigDir); err != nil {
+		if len(opt.configPath) == 0 {
```

Comment:
We should not do this -- always expect a user to give the flag and error if not.

Reply:
For seamless roll-out, we need a new PR first that adds the flag but does not validate that it is set. Then, we add a PR to add the flag to the job. Then, we land this PR, where we enforce it is set and use it

Reply:
On it now.

Reply:
Done.